modules/ocf_irc/templates/inspircd.conf.erb

```diff
@@ -13,6 +13,7 @@
 <module name="m_sha256.so">
 <module name="m_sqloper.so">
 <module name="m_ssl_gnutls.so">
+<module name="m_cap.so">
```

Comment:
why was this added?

Reply:
Seems to be the capability negotiation module. I was wondering about it too.

Reply:
SASL won't work otherwise. It's part of the ircv3 spec (SASL is ircv3, although everyone supports it). Your client will do use it to see available ircv3 features the server supports. You can do this in your client with `/quote CAP LS`

https://ircv3.net/specs/extensions/sasl-3.2

Reply:
Awesome, sorry I missed that.